docs/sources/shared/configuration.md

```diff
@@ -5378,6 +5378,18 @@ bloom_shipper:
     # cache before they get purged.
     # CLI flag: -bloom.metas-lru-cache.ttl
     [ttl: <duration> | default = 1h]
+
+  memory_management:
+    # One of: simple (simple heap allocations using Go's make([]byte, n) and no
+    # re-cycling of buffers), dynamic (a buffer pool with variable sized buckets
+    # and best effort re-cycling of buffers using Go's sync.Pool), fixed (a
+    # fixed size memory pool with configurable slab sizes, see mem-pool-buckets)
+    # CLI flag: -bloom.memory-management.alloc-type
+    [bloom_page_alloc_type: <string> | default = "dynamic"]
```

Comment:
Is this something we want to expose to users? Or do we want to have it to run some experiments and will later remove it?

Reply:
I think this must be configurable, but it's kind of an "expert setting". We could hide it from the docs